tests/test_const_name.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+import itertools
+
+import pytest
+
+from pybind11_tests import const_name as m
+
+
+@pytest.mark.parametrize(
+    "func, selector_expected",
+    itertools.product(
```

Comment:
If you parameterize `func`, and `selector_expected` separately, it will implicitly do an itertools.product between them.

Reply:
Done. Thanks!